build/Packages.props

```diff
@@ -7,7 +7,7 @@
         <MicrosoftTestPackageVersion>17.0.0</MicrosoftTestPackageVersion>
         <MSBuildPackageVersion>17.0.0</MSBuildPackageVersion>
         <NuGetPackageVersion>6.0.0</NuGetPackageVersion>
-        <RoslynPackageVersion>4.2.0-1.22074.8</RoslynPackageVersion>
+        <RoslynPackageVersion>4.2.0-1.22108.11</RoslynPackageVersion>
```

Comment:
Since the ExternalAccess libraries are not publish to NuGet there is no 4.2.0-1.final version to reference, instead use the corresponding rolling build.